src/metabase/api/database.clj

```diff
@@ -220,12 +215,8 @@
                                             include-tables?))]
     {:data  (or (dbs-list :include-tables?                  include-tables?
                           :include-saved-questions-db?      include-saved-questions-db?
-                          :include-saved-questions-tables?  include-saved-questions-tables?
-                          :limit                            offset-paging/*limit*
-                          :offset                           offset-paging/*offset*)
+                          :include-saved-questions-tables?  include-saved-questions-tables?)
                 [])
-     :limit  offset-paging/*limit*
-     :offset offset-paging/*offset*
      :total  (db/count Database)}))
```

Comment:
should this just count the `dbs-list` above? no reason to issue another query.

Also, if this is no longer paginated, do we need/want to revert back to just a vector of databases rather than the `{:data [] :total N}` shape?

Reply:
FE can have either / or. It makes it easier to add it back in when we actually do want pagination finally